suite/tests/api/ir_x86_3args_avx512_evex_mask.h

```diff
@@ -545,16 +545,28 @@ OPCODE(vcvtps2pd_zhik7zhi, vcvtps2pd, vcvtps2pd_mask, X64_ONLY, REGARG(ZMM16), R
        REGARG(ZMM16))
 OPCODE(vcvtpd2ps_xlok0ld, vcvtpd2ps, vcvtpd2ps_mask, 0, REGARG(XMM0), REGARG(K0),
        MEMARG(OPSZ_16))
+OPCODE(vcvtpd2ps_xlok0bcst, vcvtpd2ps, vcvtpd2ps_mask, 0, REGARG(XMM0), REGARG(K0),
```

Comment:
Could we also enable the binutils tests that cover these at the same time?

Reply:
Perhaps, though it'd be rather tedious.

Reply:
Just to clarify: we'd want to enable them in any case eventually: you mean it would be a lot of work to enable a subset of them and you plan to enable all of them all at once later?

Reply:
Right